<launch>
    <node pkg = "ganzhi3"     type="kdtree"       name = "points_nihe"                  output = "screen" /> 
    <node pkg = "ganzhi3"     type="plane"         name = "nihe_plane"                  output = "screen" /> 
    <node pkg = "ganzhi3"     type="scanID"       name = "lidar_pub_center"     output = "screen" /> 
    <node pkg="rviz"                type="rviz"             name="rviz"         args="-d /home/zh/catkin_ws/src/ganzhi3/rviz/lidar_camera.rviz"      required="true" />

    <!-- can通信 -->
    <!-- <node pkg = "ganzhi3"     type="test1"        name = "Lasersub"             output = "screen" />  -->

    <!-- <include file = "$(find zed_wrapper)/launch/zed.launch" />  --> 
    <!-- <include file = "$(find handeye-calib)/launch/aruco_start_usb_cam.launch" /> -->
    <!-- <include file = "$(find rslidar_sdk)/launch/start.launch" /> -->
</launch>
